appservice/src/appSettings/confirmOverwriteSettings.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { DialogResponses } from "vscode-azureextensionui";
+import { ext } from "../extensionVariables";
+import { localize } from "../localize";
+
+export async function confirmOverwriteSettings(sourceSettings: { [key: string]: string }, destinationSettings: { [key: string]: string }, destinationName: string): Promise<void> {
```

Comment:
Lol turns out you already copied this code over: https://github.com/microsoft/vscode-azuretools/pull/357

Reply:
Oh right, I just moved it in this PR. lol.